### ISSUE TITLE
f-droid version

### ISSUE THREAD

**Issue by darkknight7777777** [KhalidWar/anonaddy#63]
Are there any plans to have an f-droid version? I like the app but would prefer a version on that repository.

[https://f-droid.org/](url)

**Comment by KhalidWar** [comment]
Hi DarkKnight,

Yes, I tried publishing AddyManager there before but the process was confusing. So, I abandonned the idea.

I'm adding it to my todo and revisit the idea again.

Thanks for your suggestion.

**Comment by shuvashish76** [comment]
We do have Stjin's version of AnonAddy on F-Droid but good to have multiple user choices :)
https://anonaddy.com/faq/#is-there-an-android-app

RFP: https://gitlab.com/fdroid/rfp/-/issues/2198

**Comment by licaon-kter** [comment]
@KhalidWar what part of the process you've fo

**Comment by KhalidWar** [comment]
@licaon-kter publishing process. I'm stil looking into it.

**Comment by licaon-kter** [comment]
There's nothing to look into.

The easy route is to open a RFP, and wait for contributors to help.

Or, look at any other app (you don't use any from F-Droid? :)) recipe and try to open a MR to add your `metadata/yourappid.yml` ;)

**Comment by IzzySoft** [comment]
@KhalidWar there's already [an open RFP](https://gitlab.com/fdroid/rfp/-/issues/2198) waiting for you to add metadata (description, screenshots etc) here using Fastlane structures. Any chance you could do that? You're welcome to use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for guidance.

**Comment by KhalidWar** [comment]
> @KhalidWar there's already [an open RFP](https://gitlab.com/fdroid/rfp/-/issues/2198) waiting for you to add metadata (description, screenshots etc) here using Fastlane structures. Any chance you could do that? You're welcome to use my [Fastlane Cheat Sheet](https://gitlab.com/-/snippets/1895688) for guidance.

Thanks for sending this over.

**Comment by IzzySoft** [comment]
So any chance? Didn't hear back from you.

**Comment by IzzySoft** [comment]
@KhalidWar still no word from you – the RFP is waiting for your action. The usual process is it would now enter the closure phase and, should there be no further action within 4 weeks, the RFP would be closed unprocessed. I guess that's not what you'd like, so I postpone that stage for a last time to give you a better chance. But when the timer rings next time, the RFP will enter closure phase – so it would be good could you act now :wink:

**Comment by shuvashish76** [comment]
@KhalidWar  #74 for F-Droid metadata.

**Comment by xanoni** [comment]
I was really surprised to not see an F-Droid version of this app. Would also strongly encourage publishing it there.

**Comment by Gitchick57** [comment]
I agree.  I prefer to download / update apps via GitHub or f-droid as much as possible.



Apr 16, 2024, 5:33 AM by ***@***.***:

**Comment by licaon-kter** [comment]
last state is (waiting for feedback) https://gitlab.com/fdroid/rfp/-/issues/2198#note_1504391690
